master/internal/rm/multirm/multirm.go

```diff
@@ -234,12 +234,31 @@ func (m *MultiRMRouter) GetJobQ(rpName rm.ResourcePoolName) (map[model.JobID]*sp
 func (m *MultiRMRouter) GetJobQueueStatsRequest(req *apiv1.GetJobQueueStatsRequest) (
 	*apiv1.GetJobQueueStatsResponse, error,
 ) {
-	resolvedRMName, err := m.getRM(rm.ResourcePoolName(req.ResourcePools[0]))
+	filteredRMs := make(map[rm.ResourceManager]bool)
+	for _, rName := range req.ResourcePools {
+		resolvedRMName, err := m.getRM(rm.ResourcePoolName(rName))
+		if resolvedRMName != "" && err != nil {
+			return nil, err
+		} else if resolvedRMName != "" {
+			filteredRMs[m.rms[resolvedRMName]] = true
+		}
```

Comment:
```suggestion
		resolvedRMName, err := m.getRM(rm.ResourcePoolName(rName))
		if err != nil {
			return nil, err
		}
		filteredRMs[m.rms[resolvedRMName]] = true
```

Is this change incorrect? 

The way the code is written is weird to me validating the output of m.getRM

Reply:
Following up with you in slack with some test case examples